plugins/bazaar/src/components/ProjectCard/ProjectCard.tsx

```diff
@@ -37,39 +38,48 @@ type Props = {
   height?: string;
 };
 
-const useStyles = makeStyles({
-  statusTag: {
-    display: 'inline-block',
-    whiteSpace: 'nowrap',
-    marginBottom: '0.8rem',
-  },
-  description: {
+type StyleProps = {
+  height?: string;
+};
+
+const useStyles = makeStyles((theme: BackstageTheme) => ({
+  description: (props: StyleProps) => ({
+    height: props.height === '7rem' ? '3.7rem' : '10.2rem',
```

Comment:
This is very oddly specific 😅, I'd say you need to figure out a different way to size this, maybe a fixed size could be alright? Otherwise I think it's best to refactor the internal components to use something like `'small' | 'large'` to set the size.

Reply:
Thanks for your comments! You are right about that it was too specific 😆  I took a closer look and think it's a little bit better now? I took your advice and use `small`and `large` as props.

Reply:
> 

I implemented functionality for avoiding the cut. But since the user can just click on the card and open the full text I  would say too much extra logic for such a small thing since. Also since it's an overview.  In the end the whole look of the overview also turned out a bit too inconsistent with different looking cards? :slightly_smiling_face: Screenshot below from one card that has overflow and one that fits perfectly. 

<img width="459" alt="Skärmavbild 2022-12-20 kl  22 29 30" src="https://user-images.githubusercontent.com/76013501/208931563-905df9a2-1a11-4324-a56b-ea2a24656255.png">